### PR TITLE
Add some workarounds in README

### PR DESCRIPTION
**Pull request by DucNg**
Add instruction for workarounds if HOOK doesn't work out of the box.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,10 +22,28 @@ If you are a **Manjaro** user and installed your system using their setup assist
...
  1. Download or clone this repository to your machine and change into it
  2. Run `makepkg` to install this hook
- 4. Add `bluetooth` to your HOOKS array in `/etc/mkinitcpio.conf` before `encrypt` and after `keyboard`
-    **Example:** HOOKS="base udev autodetect modconf keyboard keymap bluetooth block encrypt lvm2 filesystems fsck"
+ 3. Add `bluetooth` to your HOOKS array in `/etc/mkinitcpio.conf` before `encrypt` and after `keyboard`
+ 
+    **Example:** `HOOKS=(base udev autodetect modconf keyboard keymap bluetooth block encrypt lvm2 filesystems fsck)`
+ 4. **(if needed)** Add any file, binary, module that your keyboard or bt adapter might need
+ 
+    **Exemple:**
```
Comment (irreleph4nt) [inline README.md]: Can you please correct spelling to English? i.e. "Example", not "Exemple"

File: README.md [hunk 2]
```diff
@@ -22,10 +22,28 @@ If you are a **Manjaro** user and installed your system using their setup assist
...
+    FILES=(/usr/lib/firmware/intel/ibt-20-1-3.sfi) 
+    MODULES=(usbhid xhci_hcd)
+    ```
  5. Rebuild your initramfs with `mkinitcpio`
  6. Upon next reboot, your keyboard should become operational automatically just before cryptsetup asks for your passphrase
+
+## Troubleshooting
+If you have trouble finding what module, file, binary you need for your keyboard or bt adapter, try looking for errors in dmesg and journalctl related to bluetooth.
+
+**Exemple:**
```
Comment (irreleph4nt) [inline README.md]: Same as above, please correct spelling.

**Review by irreleph4nt — CHANGES_REQUESTED** [review]
Thank you for your contribution. With a few easy fixes I am happy to merge this.

**Comment by DucNg** [comment]
Of course, here you go!

**Review by irreleph4nt — APPROVED** [review]
Thank you, looks good to merge to me.

**Pull request merged**